js/app.js

```diff
@@ -1 +1,316 @@
+		|| (boardArray[3] == "O" && boardArray[4] == "O" && boardArray[5] == "O")) {
+		if(boardArray[3] == "X") {
+			alert("X wins!"); }
+			else {
+				alert("O wins!");
+			}
+		win = 1;
+	} else if ((boardArray[6] == "X" && boardArray[7] == "X" && boardArray[8] == "X")
+		|| (boardArray[6] == "O" && boardArray[7] == "O" && boardArray[8] == "O")) {
+		if(boardArray[6] == "X") {
+			alert("X wins!"); }
+			else {
+				alert("O wins!");
+			}
+		win = 1;
+	} else if ((boardArray[0] == "X" && boardArray[3] == "X" && boardArray[6] == "X")
+		|| (boardArray[0] == "O" && boardArray[3] == "O" && boardArray[6] == "O")) {
+		win = 1;
+		if(boardArray[0] == "X") {
+			alert("X wins!"); }
+			else {
+				alert("O wins!");
+			}
+	} else if ((boardArray[1] == "X" && boardArray[4] == "X" && boardArray[7] == "X")
+		|| (boardArray[1] == "O" && boardArray[4] == "O" && boardArray[7] == "O")) {
+		win = 1;
+		if(boardArray[1] == "X") {
+			alert("X wins!"); }
+			else {
+				alert("O wins!");
+			}
+	} else if ((boardArray[2] == "X" && boardArray[5] == "X" && boardArray[8] == "X")
+		|| (boardArray[2] == "O" && boardArray[5] == "O" && boardArray[8] == "O")) {
+		if(boardArray[2] == "X") {
+			alert("X wins!"); }
+			else {
+				alert("O wins!");
+			}
+		win = 1;
+	} else if ((boardArray[0] == "X" && boardArray[4] == "X" && boardArray[8] == "X")
+		|| (boardArray[0] == "O" && boardArray[4] == "O" && boardArray[8] == "O")) {
+		if(boardArray[0] == "X") {
+			alert("X wins!"); }
+			else {
+				alert("O wins!");
+			}
+		win = 1;
+	} else if ((boardArray[2] == "X" && boardArray[5] == "X" && boardArray[8] == "X")
+		|| (boardArray[2] == "O" && boardArray[5] == "O" && boardArray[8] == "O")) {
+		if(boardArray[2] == "X") {
+			alert("X wins!"); }
+			else {
+				alert("O wins!");
+			}
+		win = 1;
+	} else if ((boardArray[2] == "X" && boardArray[4] == "X" && boardArray[6] == "X")
+		|| (boardArray[2] == "O" && boardArray[4] == "O" && boardArray[6] == "O")) {
+		if(boardArray[2] == "X") {
+			alert("X wins!"); }
+			else {
+				alert("O wins!");
+			}
+		win = 1;
+	} else if (count == 9) {
+		alert("No one wins!");
+		win = 1;
+	}
+
+	if (win == 1) {
+		spaces[0].removeEventListener('click', placeXO_0);
+		spaces[1].removeEventListener('click', placeXO_1);
+		spaces[2].removeEventListener('click', placeXO_2);
+		spaces[3].removeEventListener('click', placeXO_3);
+		spaces[4].removeEventListener('click', placeXO_4);
+		spaces[5].removeEventListener('click', placeXO_5);
+		spaces[6].removeEventListener('click', placeXO_6);
+		spaces[7].removeEventListener('click', placeXO_7);
+		spaces[8].removeEventListener('click', placeXO_8);
+	}	
+}
+
+// Show Turn Count
+
+var turnNumber = document.getElementById('turn');
+
+function displayTurn() {
+	turnNumber.textContent = "Turn Number: " + (count + 1);
+}
+
+// Resets the game board
+
+document.getElementById('reset').addEventListener('click', function() {
+	zero.style.visibility = "hidden";
```

Comment:
Right here in order to reduce repeating code, you could set all of these to the same class. Then in your css give that class visibility:hidden